tests/test_watch_cli.py

```
@@ -114,3 +137,44 @@ def failing_watch(*_args, **_kwargs):
 
     with pytest.raises(RuntimeError, match="watchfiles unavailable"):
         run_watch_cli(args, env)
+
+
+def test_watch_passes_flags(monkeypatch, tmp_path):
```

Ruff (0.15.9)

[warning] Unused function argument: `monkeypatch`

(ARG001)